CONTRIBUTING.md

```diff
@@ -25,35 +25,68 @@ There are only a few rules and advices we want to give to you:
     - [Documenting new features](#documenting-new-features)
     - [Sample Templates](#sample-templates)
 
-- [Committing code](#commiting-code)
-    - [Committing guidelines](#commiting-guidelines)
-
 ## Advice for new contributors
 
-### First steps
-* Clone/fork our github/gitlab repository
-~~~
-git clone https://cst.version.fz-juelich.de/jupedsim/jpscore.git
-~~~
+### Building from source
+The build is tested on Ubuntu 18.04 with gcc-8 / clang-8 and on macOS Mojave
+10.14.5 and apple clang.
 
-* Change to the developement branch and create a branch with your feature.
-~~~
-git checkout developement 
-git checkout -b feature_name
-~~~
+#### Requirements
+For the compilation of JuPedSim core you need a C++17 capable compiler and standard library.
+
+Required:
+* boost (>= 1.65)
+* libomp (if you want to use OpenMP with Apple LLVm or Clang on Linux)
+* cmake (>= 3.1)
 
-* Assuming you are in the jpscore folder type 
+Recommended:
+* ninja-build
+
+The following dependencies could be installed system wide or will be installed locally using a script:
```

Comment:
using the install [script](https://github.com/JuPedSim/jpscore/blob/develop/scripts/setup-deps.sh)